### PR TITLE
fix(cb2-9269): IVA test types missing for LGV and PSV Provisional Tech Records

### PR DESCRIPTION
**Pull request by owen-corrigan-bjss**
CB2-9269

IVA test types missing for LGV and PSV Provisional Tech Records
[link to ticket number]()

## Checklist

- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: tests/resources/test-types.json [hunk 1]
```diff
@@ -9051,7 +9086,7 @@
                     "forVehicleSubclass": null,
                     "forVehicleWheels": 3,
                     "defaultTestCode": "ea3",
-                    "linkedTestCode": null
```
Comment (shivangidas) [inline tests/resources/test-types.json:9054]: did you mean to delete this?

Comment (owen-corrigan-bjss) [reply tests/resources/test-types.json:9054]: No I didn't I'll put it back in

**Review by kcarr-bjss — APPROVED** [review]

**Review by kpearmanbjss — APPROVED** [review]

**Review by andrewf-bjss — APPROVED** [review]
YHL and YNZ?

**Comment by andrewf-bjss** [comment]
> YHL and YNZ?
YHL is there, I just missed it - thanks @owen-corrigan-bjss

**Comment by kpearmanbjss** [comment]
From what we could see YNZ just doesn't exist at all on the Taxonomy so if it's the same Taxonomy as VTA is using then there's either a mistake on the sheet or that's a test that just isn't supported by either system yet?

**Comment by andrewf-bjss** [comment]
Confirmation from Karen that YNZ is a Northern Ireland code and not required  :)

**Pull request merged**